src/js/burials/config/form.js

```diff
@@ -67,6 +79,66 @@ const formConfig = {
           }
         }
       }
+    },
+    veteranInformation: {
+      title: 'Veteran Information',
+      pages: {
+        veteranInformation: {
+          title: 'Deceased Veteran information',
+          path: 'veteran-information',
+          uiSchema: {
+            'ui:title': 'Deceased Veteran information',
+            veteranFullName: fullNameUI,
+            'view:veteranId': personId.uiSchema(
+              'veteran',
+              'view:veteranId.view:noSSN',
+              'I don’t have the Veteran’s Social Security number'
```

Comment:
I believe the guidance we've gotten before from @peggygannon and/or the previous copy person was that "number" should be lower case.